### DISCUSSION TITLE
PrimeNG Apollo Template and Primeblocks application navigation bar

### DISCUSSION THREAD

**Discussion by KasGeDev** [primefaces/community#132] [PrimeNG Templates] [answered]
I'd like to customize the[ PrimeNG Apollo Template](https://www.primefaces.org/apollo-ng), and to do that, I want to use components from PrimeBlocks, specifically the [Application Navbar](https://blocks.primeng.org/#/application/navbar) or [Application SidebarLayout](https://blocks.primeng.org/#/application/sidebarlayout)

My goal is to make the top navbar fixed in its position, and I want the sidebar navigation to function in the same way (with options like static, overlay, slim, etc.) as it does in the Apollo template.

Example:

![image](https://github.com/primefaces/community/assets/146329817/c66cfe61-48de-4110-8e32-b479b8cabb5c)

Is it possible to use them together?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Yes, it's possible to use PrimeBlocks in any PrimeNG template. But you need to adapt the logic to the selected block to behave as you want and it's up to you.
